signup.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html><head>
+  <style>
+  button {
+      background-color: #4CAF50;
+      color: white;
+      margin: 8px 0;
+      border: none;
+      cursor: pointer;
+      padding: 7px 40px;
+      width: 15%;
+  }
+  
+  button:hover {
+      opacity: 0.8;
+  }
+  body {
+    background-color: #BDB76B;
+}</style>
+    
+
+<title>Maintainance Tracker</title>
+</head>
+<body>
+    <form  >
+  
+    <h1>Sign Up</h1>
+    <p>Please fill in this Your details to create an account.</p>
+
+    <table style = "width:50%">
+ 
+        <tr>
+          <td style="font-size: 130%"><label>First Name</label> </td>
+          <td ><input type="text"></td> 
+
+        </tr>
+        <tr>
+            <td style="font-size: 130%"><label>Last Name</label> </td>
```

Comment:
Inline styles are not good, Just imagine if you had to change each font size property.
Consider using  CSS class selectors to enable sharing styles